packages/testing/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@kiltprotocol/testing",
+  "private": true,
+  "version": "0.26.0-2",
+  "description": "",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "types": "./lib/cjs/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "files": [
+    "lib/**/*"
+  ],
+  "scripts": {
+    "updateMetadata": "node ./scripts/fetchMetadata.js -o './src/mocks/metadata/spiritnet.json' -e 'https://spiritnet.api.onfinality.io/public-http'",
+    "clean": "rimraf ./lib",
+    "build": "yarn clean && yarn build:ts",
```

Comment:
does this need to be built?

Reply:
Pretty sure it does, code that is imported from another package (i.e. testing package) would always import from the path that's in the package.json, right? And we probably can't point to ts code there. The only alternative would be to write js instead I assume.
If we do not build, tests currently fail with `Cannot find module '@kiltprotocol/testing' from '<test file>'`.